pkg/kube/factory.go

```diff
@@ -31,9 +29,6 @@ type Factory interface {
 	// ToRawKubeConfigLoader return kubeconfig loader as-is
 	ToRawKubeConfigLoader() clientcmd.ClientConfig
 
-	// DynamicClient returns a dynamic client ready for use
-	DynamicClient() (dynamic.Interface, error)
```

Comment:
Since it's failing CI due to deprecated methods, can you restore this on the Interface. We may not need it right now but we would like to minimize the Interface change for now.

Reply:
I restored the `DynamicClient()` method on the interface like you asked me, but I don’t get the link with the error in the CI (deprecation of the `wait.PollImmediateUntil` function)?